test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigratePackageDependencies.cs

```diff
@@ -29,6 +29,92 @@ public void It_migrates_basic_PackageReference()
         }
 
         [Fact]
+        public void It_migrates_asp_project_dependency_tools_packages()
```

Comment:
I have tests for this already. It is in a test class called GivenThatIWantToMigrateAspNetTools: https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateAspNetTools.cs